pycine/file.py

```diff
@@ -45,6 +48,67 @@ def read_chd_header(chd_file):
 
     return header
 
+def taggedBlock(cine_file, header):
+    #with help of https://raw.githubusercontent.com/soft-matter/pims/master/pims/cine.py
```

Comment:
Unless you really took code from pims, I guess we can skip that link

Reply:
I inspired me in converting the timestamp data into second values. So I not really used code, I rahter coded is similar.